src/audio/google/google_ctc_audio_processing.c

```diff
@@ -408,6 +431,7 @@ static int ctc_process(struct processing_module *mod,
 	}
 
 	cd->ctc_func(cd, source, sink, &input_buffers[0], &output_buffers[0], frames);
+
```

Comment:
drive-by, and although in general I personally do prefer an empty line before `return`, this is inconsistent with your above changes, where there's no such line